tests/debugger/do_help.expected

```diff
@@ -4,8 +4,8 @@ Stopped at tests/debugger/do_help.js:15
 
 Documented commands (type help <topic>):
 ========================================
-b          bt        delete  eval       help  next        s   
-backtrace  c         dump    exception  list  pendingdel  src 
-break      continue  e       fbreak     n     quit        step
+b          bt        delete  eval       help      n           quit  step
```

Comment:
Yes, it does. `Documented commands (type help <topic>):
========================================
b          bt        delete  eval       help      n           quit  step
backtrace  c         dump    exception  list      next        s   
break      continue  e       fbreak     memstats  pendingdel  src 

(jerry-debugger) 
`